project-folder.Rmd

```diff
@@ -147,22 +156,26 @@ automatically check the current R version is the same as we defined in `.Rprofil
 R_version <- "4.1.1"
 ```
 
-If the R version mismatch, an error message is displayed as below.
+If there is an R version mismatch, an error message is displayed as below.
 
 ```
 Error: The current R version is not the same as the current project in R4.1.1
 ```
 
-> `.Rprofile` is only for project-specific R packages.
-> A standard R package should not use `.Rprofile`.
+::: {.rmdnote}
```

Comment:
I am not familiar with this style. When shall we use it?

Reply:
Check this out: https://bookdown.org/yihui/bookdown/html.html#callout-blocks

Reply:
The `:::` thing is pandoc grammar for creating `<div>` blocks: https://pandoc.org/MANUAL.html#divs-and-spans